tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -582,3 +582,72 @@ CGPathRef newPathForRoundRect(CGRect rect, CGFloat radius) {
 
     EXPECT_TRUE(test);
 }
+
+void EXPECT_POINTEQ(CGPoint pathPoint, CGFloat x, CGFloat y) {
+    EXPECT_EQ(x, pathPoint.x);
+    EXPECT_EQ(y, pathPoint.y);
+}
+
+void EXPECT_SIZEEQ(CGSize pathSize, CGFloat width, CGFloat height) {
+    EXPECT_EQ(width, pathSize.width);
+    EXPECT_EQ(height, pathSize.height);
+}
+
+// Simple tests for the status of a CGPath during the CGPath rework into D2D.
+TEST(CGPath, CGPathD2DSimpleCreation) {
```

Comment:
D2D here is an implementation detail.  Since you are only calling public APIs, please drop th D2D prefix.  Or have an issue to go rename them later.